pyproject.toml

```diff
@@ -80,6 +85,7 @@ pytest-twisted = "<=1.13.4"
 pytest-xvfb = "1.2.0"
 twine = "1.13.0"
 pyqt5-tools = { version = "5.13.2.1.6rc1", markers = "sys_platform == 'win32'" }
+poetry-dynamic-versioning = "^0.13.1"
```

Comment:
This is the old way that we are trying to remove.
```suggestion
```

Reply:
Resolved.